### PR TITLE
admin should read/write to the correct env file

### PR DESCRIPTION
**Pull request by Joshuafolorunsho**
Will need to refactor the FileEnvService so we don't need to duplicate.

### CODE REVIEW

File: packages/admin/src/service/service.fileEnv.ts [hunk 1]
```diff
@@ -42,15 +67,19 @@ export class FileEnvService extends EnvService {
...
   async setEnvValue(key: string, value: string): Promise<void> {
     try {
-      let data = await this.readFile();
-      const regex = new RegExp(`^${key}=.*$`, 'm');
-      if (data.match(regex)) {
-        data = data.replace(regex, `${key}=${value}`);
-      } else {
-        data += `\n${key}=${value}`;
+      const adminEnvData = await this.readFile(this.filePath);
+      const arkWorkingDirectoryPath = process.env.ARK_APP_DIR as string;
```
Comment (smthomas) [inline packages/admin/src/service/service.fileEnv.ts]: We had to update this based on some NextJS issues. In the Next config file you will see that we pass it in as CONFIG_PATH. Changing to `process.env.CONFIG_PATH` should work.

**Review by smthomas — APPROVED** [review]
Overall I think this looks okay. We definitely need to come back in and rethink how to to duplicate the code. Maybe pulling it out into core is an option or a shared util package.

File: packages/admin/src/service/service.fileEnv.ts (hunk 1 above)
Comment (Joshuafolorunsho) [reply packages/admin/src/service/service.fileEnv.ts]: Updated. Curious what the issues were.

**Comment by Joshuafolorunsho** [comment]
> Overall I think this looks okay. We definitely need to come back in and rethink how to to duplicate the code. Maybe pulling it out into core is an option or a shared util package.

Leaning towards a shared util packages, but yeah. We do need to come back and think about it.

**Review by smthomas — APPROVED** [review]

**Pull request merged**